### PR TITLE
Dataset mass operations

### PR DESCRIPTION
**Pull request by flops**

### CODE REVIEW

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/charts-9817-ph-1-dataset-mass-operations-1147/dataset-dataset-editor-modify-1147/ru/">dataset.dataset-editor.modify (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/charts-9817-ph-1-dataset-mass-operations-1147/dataset-dataset-editor-modify-1147/en/">dataset.dataset-editor.modify (en)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/charts-9817-ph-1-dataset-mass-operations-1147/dataset-dataset-editor-modify-1147/ru/">dataset.dataset-editor.modify (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/charts-9817-ph-1-dataset-mass-operations-1147/dataset-dataset-editor-modify-1147/en/">dataset.dataset-editor.modify (en)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/charts-9817-ph-1-dataset-mass-operations-1147/dataset-dataset-editor-modify-1147/ru/">dataset.dataset-editor.modify (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/charts-9817-ph-1-dataset-mass-operations-1147/dataset-dataset-editor-modify-1147/en/">dataset.dataset-editor.modify (en)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/charts-9817-ph-1-dataset-mass-operations-1147/dataset-dataset-editor-modify-1147/ru/">dataset.dataset-editor.modify (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/charts-9817-ph-1-dataset-mass-operations-1147/dataset-dataset-editor-modify-1147/en/">dataset.dataset-editor.modify (en)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/charts-9817-ph-1-dataset-mass-operations-1147/dataset-dataset-editor-modify-1147/ru/">dataset.dataset-editor.modify (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/charts-9817-ph-1-dataset-mass-operations-1147/dataset-dataset-editor-modify-1147/en/">dataset.dataset-editor.modify (en)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/charts-9817-ph-1-dataset-mass-operations-1147/dataset-dataset-editor-modify-1147/ru/">dataset.dataset-editor.modify (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/charts-9817-ph-1-dataset-mass-operations-1147/dataset-dataset-editor-modify-1147/en/">dataset.dataset-editor.modify (en)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/charts-9817-ph-1-dataset-mass-operations-1147/dataset-dataset-editor-modify-1147/ru/">dataset.dataset-editor.modify (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/charts-9817-ph-1-dataset-mass-operations-1147/dataset-dataset-editor-modify-1147/en/">dataset.dataset-editor.modify (en)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/charts-9817-ph-1-dataset-mass-operations-1147/dataset-dataset-editor-modify-1147/ru/">dataset.dataset-editor.modify (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/charts-9817-ph-1-dataset-mass-operations-1147/dataset-dataset-editor-modify-1147/en/">dataset.dataset-editor.modify (en)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/charts-9817-ph-1-dataset-mass-operations-1147/dataset-dataset-editor-modify-1147/ru/">dataset.dataset-editor.modify (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/charts-9817-ph-1-dataset-mass-operations-1147/dataset-dataset-editor-modify-1147/en/">dataset.dataset-editor.modify (en)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

**Comment by robot-charts** [comment]
**i18n-check**
<details>
<summary>The following components have not been translated</summary>
<p><a href="https://weblate.gravity-ui.com/projects/datalens/charts-9817-ph-1-dataset-mass-operations-1147/dataset-dataset-editor-modify-1147/ru/">dataset.dataset-editor.modify (ru)</a><br><a href="https://weblate.gravity-ui.com/projects/datalens/charts-9817-ph-1-dataset-mass-operations-1147/dataset-dataset-editor-modify-1147/en/">dataset.dataset-editor.modify (en)</a></p>
</details>

Wait for the reviewers to check your changes in Weblate and try running github action again.

**Comment by robot-charts** [comment]
**i18n-check**
Please merge the Pull Request with the changes from Weblate into your branch.

**Review by korvin89 — APPROVED** [review]

File: src/shared/types/dataset.ts [hunk 1]
```diff
@@ -324,5 +324,7 @@ export interface DatasetApiError {
     };
 }
 
+export type DatasetSelectionMap = Record<DatasetField['guid'], true>;
```
[the remark this answers is not in the document]
Comment (flops) [reply src/shared/types/dataset.ts:327]: It more about syntax, that we can see by which field map built

**Pull request merged**